<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewProjectWizard</class>
 <widget class="QWizard" name="NewProjectWizard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>362</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Project</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/icons/folder_add.png</normaloff>:/icons/folder_add.png</iconset>
  </property>
  <widget class="QWizardPage" name="wizardPage1">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="headerLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="icons.qrc">:/icons/folder_add.png</pixmap>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>New Project</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="generalGroup">
      <property name="title">
       <string>General</string>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
       </property>
       <property name="horizontalSpacing">
        <number>6</number>
       </property>
       <property name="verticalSpacing">
        <number>6</number>
       </property>
       <property name="margin">
        <number>9</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="projectLocationLabel">
         <property name="text">
          <string>Project Location:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLineEdit" name="projectLocationEdit"/>
         </item>
         <item>
          <widget class="QPushButton" name="projectLocationButton">
           <property name="text">
            <string>Select Folder</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="projectNameLabel">
         <property name="text">
          <string>Project Name:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="projectNameEdit"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="informationLabel">
         <property name="text">
          <string>Target:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLabel" name="createdPathLabel">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QCheckBox" name="updateDefaultLocationCheckBox">
         <property name="text">
          <string>Use as default project location</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>120</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>projectNameEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>NewProjectWizard</receiver>
   <slot>updateProjectLocation()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>399</x>
     <y>141</y>
    </hint>
    <hint type="destinationlabel">
     <x>466</x>
     <y>207</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>projectLocationButton</sender>
   <signal>pressed()</signal>
   <receiver>NewProjectWizard</receiver>
   <slot>selectProjectLocation()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>377</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>459</x>
     <y>112</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>updateProjectLocation()</slot>
  <slot>selectProjectLocation()</slot>
 </slots>
</ui>
